### PR TITLE
Harmonize user docs

### PR DESCRIPTION
**Pull request by NobodysNightmare**
## OpenProject work package
- https://community.openproject.org/work_packages/18094
## Remark

I was brave enough to **implement** the missing link to the already existing delete action.
Note that I am a ruby newbie and my local setup does not allow me to run the tests... So you should be really careful reviewing my code changes ;-)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://feature-harmonize-user-docs.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: doc/apiv3-documentation.apib [hunk 1]
```diff
@@ -1991,9 +2003,9 @@ This endpoint lists the types that are *available* in a given project.
                 "lastName": "Sheppard",
                 "mail": "shep@mail.com",
                 "avatar": "https://gravatar/avatar",
+				"status": "active",
```
Comment (ulferts) [inline doc/apiv3-documentation.apib]: indentation

File: doc/apiv3-documentation.apib [hunk 2]
```diff
@@ -1958,9 +1964,15 @@ This endpoint lists the types that are *available* in a given project.
...
 | avatar      | URL to user's avatar                                      | String   |             | READ                 |
+| status      | The current activation status of the user (see below)     | String   |             | READ                 |
 | createdAt   | Time of creation                                          | DateTime |             | READ                 |
 | updatedAt   | Time of the most recent change to the user                | DateTime |             | READ                 |
 
+The `status` of a user can be one of:
+
+* `active` - the user can act normally
+* `registered` - the user just registered to the instance, activation by an administrator is required
+* `locked` - the user was locked and can't login
```
Comment (ulferts) [inline doc/apiv3-documentation.apib]: Just some questions regarding wording here:
- what do you mean by "can act normally"? I wouldn't want to use the word "normally" to describe my behaviour at the best of times.
  - Proposal: "The user can log in with the account" 
- `activation by an administrator is required`. He could also be in the first phase of registering himself. He would then have to activate his account by following the link in the e-mail.
  - Proposal: "The user can not log in yet but will be able to once the registration is completed."
- `the user was locked and can't login` the account is still locked so "was" seems wrong to me.

**Comment by ulferts** [comment]
LGTM except for the minor things I annotated.

File: doc/apiv3-documentation.apib (hunk 1 above)
Comment (NobodysNightmare) [inline doc/apiv3-documentation.apib]: Thanks for that one... I managed to introduce >1000 tabs with this PR ^^

Editor settings are now fixed again.

**Pull request merged**